conf/defaults.ini

```diff
@@ -1146,6 +1146,9 @@ explore2Dashboard = true
 # Experimental Command Palette
 commandPalette = true
 
+# Use dynamic labels in CloudWatch datasource
+cloudWatchDynamicLabels = false
```

Comment:
Only went looking for this because i got a conflict elsewhere, but I'm sure feature flags are off by default, so we probably don't need this extra value in the config?

Reply:
Oh yeah I see what you mean. 
Last step of this epic (hopefully completed next week) would be to set this to true. Would that be ok to leave it in place for now?